qiskit_nature/circuit/library/ansatzes/succd.py

```diff
@@ -37,18 +39,13 @@
 
 class SUCCD(UCC):
     """The SUCCD Ansatz.
-
```

Comment:
Please revert the removal of these deleted lines

Reply:
Pinging this comment again. Please restore this entire file to the status of `main`